src/Forms/GridField/GridFieldFilterHeader.php

```diff
@@ -76,6 +76,12 @@ class GridFieldFilterHeader implements GridField_URLHandler, GridField_HTMLProvi
      */
     protected $updateSearchFormCallback = null;
 
+    /**
+     * The name of the default search field
+     * @var string|null
+     */
+    protected $searchField = null;
```

Comment:
I would make this private with the expectation that child class that need to interact with it should use/override the accessor/mutator.

```suggestion
    private $searchField = null;
```

Reply:
My gut feeling was private, but the rest of the api isn't private, so it feels awkward to me. But I'll defer to you here.